mongodb/6.2/modules/ROOT/pages/mongodb-connector-reference.adoc

```diff
@@ -38,8 +38,8 @@ Release Notes: xref:release-notes::connector/mongodb-connector-release-notes-mul
 | Maximum Staleness Seconds a| Number |  Specifies, in seconds, how stale a secondary node on a MongoDB cluster can be before the client stops using it for read operations. |  -1 |
 | Read Preference Tags a| Array of <<ReadPreferenceTag>> |  If replica set members are associated with tags, you can specify a tag set (array of tag specification documents) in the read preference to target those members. |  |
 | Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  |
-| Write Concern Acknowledgment a| String |  Requests acknowledgment that the write operation has propagated to the specified number of MongoDB instances. You can write a number of instances n>0 or use a value from the list. |  |
-| Write Concern Timeout a| Number |  This option specifies a time limit, in milliseconds, for the Write Concern. The option applies to Write Concern Acknowledgment values greater than 1. |  |
+| Write Concern Acknowledgment a| String | Requests acknowledgment that the write operation has propagated to the specified number of MongoDB instances. You can write a number of instances n>0 or use a value from the list. |  |
+| Write Concern Timeout a| Number |  Specifies a time limit, in milliseconds, for the Write Concern Acknowledgement and applies to values greater than `1` |  |
```

Comment:
The "and applies to values greater than '1' seems disconnected from the other part of the sentence. I would just make two sentences: 

 Specifies a time limit, in milliseconds, for the Write Concern Acknowledgement. The value must be greater than `1`.